libraries/types/dag_block/include/dag/dag_block.hpp

```diff
@@ -7,6 +7,7 @@ namespace taraxa {
 
 using std::string;
 using VdfSortition = vdf_sortition::VdfSortition;
+using estimations_vec_t = std::vector<uint64_t>;
```

Comment:
what is the advantage of using this alias ? In terms of length it the same as `std::vector<uint64_t>`